src/dcurl.h

```diff
@@ -23,16 +23,32 @@
  * easily.
  */
 
+/*! The default value of the broker hostname */
+#define DEFAULT_BROKER_HOST "localhost"
+
+/**
+ * A structure representing the configuration of the initialization.
+ */
+typedef struct {
+    char *broker_host; /**< The broker hostname used in the remote mode */
+} init_config;
```

Comment:
Don't name it `init_config`. Instead, use `dcurl_confog`.

Reply:
Done.